app/javascript/packages/document-capture/components/document-side-acuant-capture.jsx

```diff
@@ -44,7 +44,8 @@ function DocumentSideAcuantCapture({
   className,
 }) {
   const error = errors.find(({ field }) => field === side)?.error;
-  const { changeStepCanComplete } = useContext(FormStepsContext);
```

Comment:
Removing this isn't good enough for anything beyond prototyping, `changeStepCanComplete` should be `false` when there's red error text showing, then `true` when the user uploads a new image and the red text disappears. That's what the code in `main` currently is doing.